roles/foreman_installer_devel_scenario/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: 'Install git'
+  package:
+    name: git
+    state: present
+
+- name: 'Copy scenario files'
+  copy:
+    src: "{{ role_path }}/files/{{ item }}"
+    dest: /etc/foreman-installer/scenarios.d
+  with_items:
+    - katello-devel.yaml
+    - katello-devel-answers.yaml
+
+- name: 'Remove current puppet-katello_devel'
+  file:
+    state: absent
+    path: /usr/share/katello-installer-base/modules/katello_devel
+
+- name: 'Clone latest puppet-katello_devel'
+  git:
+    repo: https://github.com/theforeman/puppet-katello_devel
```

Comment:
Is there a way to specify a PR to checkout, or will that still be handled by the installer?

Reply:
I'm not really a fan of this, why do we need to do this?

Reply:
Nevermind, I guess it is nicer if you want a faster turnaround time and not have to wait for a new package.  I think the ability to check out a PR needs to be restored though

Reply:
Handled the same as before with the installer module_prs

Reply:
Oh right :man_facepalming: Disregard.